lib/context.js

```diff
@@ -350,6 +350,11 @@ api.createTermDefinition = ({
       'Invalid JSON-LD syntax; keywords cannot be overridden.',
       'jsonld.SyntaxError',
       {code: 'keyword redefinition', context: localCtx, term});
+  } else if(term.match(/@[a-zA-Z]+$/)) {
```

Comment:
Just want to find things that start with `@...` right?
```suggestion
  } else if(term.match(/^@[a-zA-Z]+$/)) {
```

Reply:
Same with other patterns below.

Reply:
Might be useful to move the regex to a common const at the top of the file.